cmd/observe/io_reader_observer_test.go

```diff
@@ -70,6 +70,82 @@ func Test_getFlowsTimeRange(t *testing.T) {
 	assert.Equal(t, io.EOF, err)
 }
 
+func Test_getFlowsLast(t *testing.T) {
+	flows := []*observer.GetFlowsResponse{
```

Comment:
rebase clash with f07dedc11f067af55c1d82f307708d5e0321e856, needs `s/observer/observerpb/g` and `s/flow/flowpb/g`.

Reply:
Ah, of course I forgot to rerun tests after rebasing. Yep will fix.